tests/integration/payload_test.js

```diff
@@ -0,0 +1,45 @@
+var integrationHelper = require('../support/integration_helper'),
+    request = require('supertest'),
+    chai = require('chai'),
+    sinon = require('sinon'),
+    sinonChai = require('sinon-chai'),
+    stubs = require('../stubs/index'),
+    blueprints = require('../blueprints/index');
+
+chai.should();
+chai.use(sinonChai);
+
+describe('Integration: Alerts Payload Endpoints', function(){
+  var clientStub, app;
+
+  before(function(){
+    clientStub = integrationHelper.clientStub;
+    var system = integrationHelper.before();
+    app = system.app;
+    clientStub = system.clientStub;
+  });
+
+  after(integrationHelper.after);
+
+  describe('POST /v1/alerts', function(){
```

Comment:
In this case since the error is not related with a specifc route but with the feature itself maybe this describe shouldnt exist